docs/software/devtools/debug/index.md

```diff
@@ -0,0 +1,11 @@
+[](){#ref-software-devtools-debug}
+# Debugging tools
+
+Learning to debug a code effectively will not only help you quickly
```

Comment:
For Markdown and git, it is best practice to have one sentence per line (turn off line wrapping).
This means that changes to different sentences are less likely to introduce conflicts, and changes will affect fewer lines.

Reply:
that's your preference, not mine.

Reply:
I will rephrase that - it is the policy of this documentation to have one line per sentence.